lib/rules/rule-empty-line-before/index.js

```diff
@@ -51,8 +51,12 @@ const rule = function(expectation, options, context) {
         return;
       }
 
-      // Ignore the first node
-      if (rule === root.first) {
+      if (
```

Comment:
Can we please move this into a `util`, e.g. `isFirstNodeOfRoot(node)` as I suspect other rules will need too.